gandi/data_domain.go

```diff
@@ -3,7 +3,7 @@ package gandi
 import (
```

Comment:
@tiramiseb @nlewo I think we should first extensively test (sadly manually) and release (or at least release) the current master, before merging this PR

Reply:
(The rc is just to block the merge, if you disagree, let me know and I'll dismiss my request change)

Reply:
Is there any reason the current master must be releases ? Couldn't the first release be with the SDK v2?

Reply:
Last time I've checked, the v2 shiped with some breaking changes, not sure if we're concerned, ofc we can read the changelog and verify if we're concerned, but since we don't have tests, no one can guarantee that it works, I think it's safer to first draft a release without these changes.

And we really need a release, just a simple tag will do for now, since we make install manually, because every time a new commit is pushed, code that depends on it is likely to break, I personally checkout to a commit before make install, which's not sustainable, since I have to keep track of the commit across various CI jobs and test all our infra every time I try to update the commit and it's also hard for new comers to tell that the safer is to find a commit that works and pin it.

I can volunteer to draft a release if you don't have time.

Reply:
I'm actually pretty confident with this change.

The [upgrade guide](https://www.terraform.io/plugin/sdkv2/guides/v2-upgrade-guide) doesn't contain any warning about this upgrade. There is a list of breaking changes in the [release notes](https://github.com/hashicorp/terraform-plugin-sdk/blob/v2.0.0/CHANGELOG.md) and i don't see anything that could impact us (other important providers no longer support terraform 0.11.).

@taoufik07 But as you suggested, i'll do more manual tests of this branch.

Reply:
Well then 🤞

Reply:
Well then 🤞